packages/renovate-config-cozy-konnector/package.json

```diff
@@ -6,22 +6,11 @@
   "renovate-config": {
     "default": {
       "extends": [
-        "config:base"
+        "cozy"
       ],
-      "timezone": "Europe/Paris",
       "minor": {
         "automerge": true
-      },
-      "packageRules": [
-        {
-          "excludePackageNames": [
-            "cozy-konnector-libs",
-            "cozy-app-publish"
-          ],
-          "schedule": "before 5am every monday"
-        }
-      ],
```

Comment:
The cozy config has almost the same rule, except that:

- it matches all `cozy-*` packages, not just the 2 specified above
- the scheduling happens every day, but still during non office hours

Reply:
For the connectors, every day is two much for 100 connectors. I prefer to keep the previous rule. But I am Ok for everything else.